mne/baseline.py

```diff
@@ -7,11 +7,19 @@
 
 import numpy as np
 
-from .utils import logger, verbose
+from .utils import logger
 
 
-@verbose
-def rescale(data, times, baseline, mode, verbose=None, copy=True):
```

Comment:
Not as easily. We want to separate the logging if possible. I was hoping this was "accidentally public" and we could assume people weren't using it, since it's not publicly documented AFAIK.